src/test/resources/study-minimal.json

```diff
@@ -28,7 +28,7 @@
     ],
     "assets":        [
       {
-        "name":   "Sample",
+        "name":   "sample",
```

Comment:
That will break my dataset connected tests.
So either don't do this or make the corresponding change to the dataset test.